prism.sh

```diff
@@ -5,25 +5,25 @@ make install
 
 API_SPEC=examples/twilio_api_v2010.yaml
 
-OUT_DIR=examples/go/rest
-rm -rf $OUT_DIR
+OUT_DIR=examples/go-client/helper/rest
+find $OUT_DIR ! -name "*_test.go" -type f -delete
```

Comment:
Just for the sake of curiosity, why the change from deleting the entire directory to the test files?